jenkins/job-configs/kubernetes-jenkins/kubernetes-djmm.yaml

```diff
@@ -47,16 +47,17 @@
             timeout: 50
             job-env: |
               export RELEASE_INFRA_PUSH=true
+              export SET_NOMOCK_FLAG=false
         - 'federation-build':
             branch: 'master'
             timeout: 50
             job-env: |
               export RELEASE_INFRA_PUSH=true
+              export SET_NOMOCK_FLAG=false
               export PROJECT="k8s-jkns-e2e-gce-federation"
               export FEDERATION=true
-              export FEDERATION_PUSH_REPO_BASE="gcr.io/k8s-jkns-e2e-gce-federation"
+              export FEDERATION_PUSH_REPO_BASE="gcr.io/djmm-federation-build"
```

Comment:
Oh right, of course PROJECT should be synced there, but yes there is no such djmm-federation-build project.  I guess I was asking if there's a FEDERATION_PUSH_REPO_BASE that is used in jenkins already that has everything set up.  Maybe there is not.  I don't see anything.

Maybe it's just fine to push things up to `k8s-jkns-e2e-gce-federation` from this test job?  @spxtr @ixdy ?